### ISSUE TITLE
Objects are not indented

### ISSUE THREAD

**Issue by muuvmuuv** [palantir/tslint#4211]
### Bug Report

- __TSLint version__: 5.11.0
- __TypeScript version__: 3.1.1
- __Running TSLint via__: CLI and Node.js API

#### TypeScript code being linted

```ts
export const test = () => {
return {
a_test: true,
some_more: "test",
};
};
```

with `tslint.json` configuration:

```json
{
    "rules": {
        "indent": [true, "spaces", 2]
    }
}
```

#### Actual behavior

Does nothing

#### Expected behaviour

```ts
export const test = () => {
  return {
    a_test: true,
    some_more: "test",
  };
};
```

**Comment by adidahiya** [comment]
duplicate of #2814
